pom.xml

```diff
@@ -367,7 +374,7 @@
         <dependency>
             <groupId>com.conveyal</groupId>
             <artifactId>osm-lib</artifactId>
-            <version>1.3.0</version>
+            <version>1.1.1</version>

```

Comment:
Why has osm-lib been changed to an older version? We see that this is somehow related to tests failing, but do you have any additional information?

Reply:
I noticed compatibility issues with osmosis-osm-binary v0.46 via osm-lib v1.3.0.  The newly released osm-lib v1.4.0 downgrades osmosis-osm-binary to v0.45 and may resolve issues.